src/components/CurrencyInputPanel/currency-panel.scss

```diff
@@ -10,6 +10,16 @@
     border: 1px solid $mercury-gray;
     background-color: $white;
     box-shadow: 0px 4px 4px 2px rgba($royal-blue, 0.05);
+
+    &--error {
+      border: 0.5px solid $error-red;
```

Comment:
Since this border width is different from the default state, the UI is jumping a bit when error state happens (by half a pixel, but small UI moves like this that affect a big portion of the layout can be distracting)

I would suggest one of the following:
- changing error border-width to 1px
- changing default border-width to 0.5px

Reply:
I prefer 0.5px for both!

Reply:
Changed to 0.5 for both